src/style/index.js

```diff
@@ -25,5 +25,8 @@ module.exports = {
   },
   get AnimatableManager() {
     return require('./animatableManager').default;
+  },
+  get ColorName() {
```

Comment:
What about keeping this class private and expose this functionality thru our Colors class.

Reply:
I can do that.

Reply:
I added `getColorName` to Colors class but I didn't turn `ColorName` to private in case someone will need the full array returned from `name()` method.

Reply:
👍